doc/getting-started/connections.rst

```diff
@@ -0,0 +1,76 @@
+Connections
+===========
+
+PyVista is a powerful tool that researchers can harness to create compelling,
+integrated visualizations of large datasets in an intuitive, Pythonic manner.
+
+PyVista is used across science and engineering disciplines by a diverse
+community to visualize data/models in 3D, generate publication-quality
+illustrations, automate analysis workflows, and build custom applications
+leveraging PyVista's 3D capabilities.
+
+Here are a few open-source projects that leverage PyVista:
+
+
+Engineering
+-----------
+
+* pyleecan_: Electrical engineering open-source software providing a user-friendly, unified, flexible simulation framework for the multiphysic design and optimization of electrical machines and drives.
+* pymapdl_: Pythonic interface to Ansys MAPDL.
+* pyprocar_: A Python library for electronic structure pre/post-processing.
+* AeroSandbox_: Plug-and-play analysis tools for aerodynamics, propulsion, structures, trajectory design, and much more.
+* Ptera_: Software for nalyzing flapping-wing flight.
+
+
+Geosciences
+-----------
+
+* GemPy_: 3D structural geological modeling software using PyVista for 3D visualization and interaction.
+* gemgis_: Spatial data processing for geomodeling.
+* gimli_: Geophysical inversion and modeling library.
+* discretize_: Discretization tools for finite volume and inverse problems.
+* SEGY-SAK_: SEGY Swiss Army Knife for Seismic Data.
+* subsurface_: Core data exchange library for subsurface science and engineering
+* Flownet_: Data-Driven Reservoir Predictions.
+* PVGeo_: Python package of VTK-based algorithms to analyze geoscientific data and models. PyVista is used to make the inputs and outputs of PVGeo's algorithms more accessible.
+* omfvista_: 3D visualization for the Open Mining Format (omf). PyVista provides the foundation for this library's visualization.
+
+
+Many More
+---------
+
+* VesselVio_: Analysis and visualization of segmented vasculature datasets.
+* pymeshfix_: Python/Cython wrapper of Marco Attene's wonderful, award-winning MeshFix_ software.
+* tetgen_: Python Interface to Hang Si's C++ TetGen Library.
+* itkwidgets_: Interactive Jupyter widgets to visualize images, point sets, and meshes in 2D and 3D. Supports all PyVista mesh types.
```

Comment:
Is it fair for us to say that itkwidgets "leverage[s] PyVista"? Frankly, itkwidgets supports `vtkPolyData` and `vtkImageData` and that means their users gain support for PyVista for free. IMO, this is perhaps misleading

Reply:
I'm not a native but I'd read "leverages PyVista" as it depending on PyVista and making explicit use of it. If it just supports the kind of VTK object that we subclass that would be something else. "Supports" like it is now? "Compatible with"?

Reply:
> "leverages PyVista" as it depending on PyVista and making explicit use of it.

Exactly my take as well. The top of this section says that all of the listed software "leverage PyVista" and I think we should use that criterion when deciding what gets included here. 

We could have another list of "Interoperable Software" - this is where I think itkwidgets would fit better. This would also give us a chance to include more software not already here.

Reply:
> We could have another list of "Interoperable Software" - this is where I think itkwidgets would fit better. This would also give us a chance to include more software not already here.

Right. Having a "works with" or "compatible with" section would be great as well.